src/OrchardCore/OrchardCore.Media.Core/Events/MediaEventHandlerBase.cs

```diff
@@ -34,5 +34,15 @@ public virtual Task MediaMovedAsync(MediaMoveContext context)
         {
             return Task.CompletedTask;
         }
+
+        public virtual Task MediaCreatingDirectoryAsync(MediaCreatingContext context)
+        {
+            return Task.CompletedTask;
+        }
+
+        public virtual Task MediaCreatedDirectoryAsync(MediaCreatedContext context)
+        {
+            return Task.CompletedTask;
+        }
```

Comment:
```suggestion
        public virtual Task MediaCreatingDirectoryAsync(MediaCreatingContext context) => Task.CompletedTask;

        public virtual Task MediaCreatedDirectoryAsync(MediaCreatedContext context) => Task.CompletedTask;
```

Reply:
I'd rather keep the file consistent, should I?

Reply:
While the method body is one line, so the change make sense

Reply:
Yes, but the rest of the file is written differently. I personally would always go for consistency, that's why I'm asking...